UnitTests/TestFixtureLifecycleAttributes/RewriteTestFixtureLifecycleAttributes.cs

```diff
@@ -31,7 +31,7 @@ public class RewriteTestFixtureLifecycleAttributes
     [TestCase ("[TestFixtureSetUp ()]", "[OneTimeSetUp ()]")]
     [TestCase ("[TestFixtureSetUp () ]", "[OneTimeSetUp () ]")]
     [TestCase ("[ TestFixtureSetUp () ]", "[ OneTimeSetUp () ]")]
-    public void TestSingleAttributeSetUpRewrites (string attribute, string expected)
+    public void TestFixtureAttributeRewriter_RewritesSingleSetupAttribute (string attribute, string expected)

```

Comment:
I just noticed 2 things:

1. Every test class needs to end with `Test`, that's not the case here.
2. Every test class should have a `[TestFixture]` attribute, I failed to notice this before but I don't think that's the case in any test class right now